cdc/changefeed.go

```diff
@@ -16,6 +16,7 @@ package cdc
 import (
 	"context"
 	"fmt"
+	"github.com/pingcap/tidb/sessionctx/binloginfo"
```

Comment:
Please follow the import rules
```
standard lib

third-party lib
```

or try to use tools like https://godoc.org/golang.org/x/tools/cmd/goimports